cmd/plugin/builder/plugin/publish-packages.go

```diff
@@ -51,26 +51,15 @@ func (ppo *PublishPluginPackageOptions) PublishPluginPackages() error {
 					continue
 				}
 
-				imageAfterLoad := fmt.Sprintf("%s/plugins/%s/%s/%s:%s", localRegistry, osArch.OS(), osArch.Arch(), pluginManifest.Plugins[i].Name, version)
 				imageToPush := fmt.Sprintf("%s/%s/%s/%s/%s/%s/%s:%s", ppo.Repository, ppo.Vendor, ppo.Publisher, osArch.OS(), osArch.Arch(), pluginManifest.Plugins[i].Target, pluginManifest.Plugins[i].Name, version)
-				log.Infof("publishing plugin 'name:%s' 'target:%s' 'os:%s' 'arch:%s' 'version:%s'", pluginManifest.Plugins[i].Name, pluginManifest.Plugins[i].Target, osArch.OS(), osArch.Arch(), version)
 
+				log.Infof("publishing plugin 'name:%s' 'target:%s' 'os:%s' 'arch:%s' 'version:%s'", pluginManifest.Plugins[i].Name, pluginManifest.Plugins[i].Target, osArch.OS(), osArch.Arch(), version)
 				if ppo.DryRun {
-					log.Infof("command: 'docker load -i %s'", pluginTarFilePath)
-					log.Infof("command: 'docker tag %s %s'", imageAfterLoad, imageToPush)
-					log.Infof("command: 'docker push %s'", imageToPush)
+					log.Infof("command: 'crane push -i %s %s'", pluginTarFilePath, imageToPush)
```

Comment:
I think:
```suggestion
					log.Infof("command: 'crane push %s %s'", pluginTarFilePath, imageToPush)
```

Reply:
This actually still does not work as a command because we first need to unzip the tar file.  But I've asked about why we do the zip/unzip as another comment first.

Reply:
I have removed the additional gzip archive-related code so the mentioned command should work now.